packages/node-opcua-server/source/opcua_server.ts

```diff
@@ -1502,7 +1503,11 @@ export class OPCUAServer extends OPCUABaseServer {
         assert(userTokenSignature.signature instanceof Buffer, "expecting userTokenSignature to be a Buffer");
 
         // verify proof of possession by checking certificate signature & server nonce correctness
-        if (!verifySignature(serverCertificate, nonce, userTokenSignature, certificate, securityPolicy)) {
+        try {
```

Comment:
Can we fix verifySignature instance to make it return false instead of throwing ?

Reply:
solved